homework.md

```diff
@@ -4,110 +4,108 @@
 
 What is 2 + 2?
 
-5
+4
 
 ## Question 2
 
 What is the opposite of Black?
 
-Purple
+White
 
 ## Question 3
 
 What is the  2 * 3 * 6 * 1 * 10 * 34?
 
-1322
+12240
 
 ## Question 4 
 
 What does HTML stand for?
 
-Hyper Tall Massive Language
+Hyper Text Markup Language
 
 ## Question 5
 
 What is CSS useful for?
 
-Making a website go faster
+The styling of HTML and Describing the presentation of a web document
 
 ## Question 6
 
 Using CSS, how would I select every `<h1>` tag on the page?
 
 ```css
-#h1 {
-
+h1 {
+  <css rules>...
 }
 ```
 
 ## Question 7
 
 What is JavaScript?
 
-A programming language used on the internet
+javascript is a programming language orginally created for the web. It is high-level programming language that is prototype based and treats objects as fitst class functions. The lanaguge has evollved significantly in the last ten years to enable object-oriented programming with syntactical 'classes' as a 'sugar' in place of the traditional need to declare prototype objects. Version EMCA5 & 6 were responsible for introducing a host of new features including lamba -like functional programming, 'class' declarations and 'promises'.
 
 ## Question 8
 
 What three problem does Git & Github solve?
 
-When people want to show off code to each other they can put it on Github
+Git allows developers to colloborate simultanusely on software projects in a distributed fashion without fear of overwriting one another's code. Each developer can clone, fork and commit code to the central repository on github and features such as the ability merge and make pull requests allow code to be reviewed. Cruciially, git allows you to 'undo' incorrect or unwanted changes to the codebase by resetting a specific commit. 
 
 ## Question 9
 
 What happens when you `fork` a repository?
 
-You delete it
+Allows you to make a copy of somebody's project or repository and to make pull requests.
 
 ## Question 10 
 
 What happens when you clone a repostory?
 
-It send it to a friend
+Cloning a repository downloads the current project hosted on github onto your laptop providing you with a local copy to work on.
 
 ## Question 11
 
 What is a Pull Request?
 
-When you send a file over the internet
-
+Allows you to request your code changes be subiited for revoew and approval before then being 'merged' .
 ## Question 12
 
 What is the Git History log?
 
-It lets you see into the future
+Provides a summary of all the changes commited by yourself and/or other developers including date and time.
 
 ## Question 13
 
 What does it meant to "push" changes to Github?
 
-You send the changes of your local repository to Github
+Uploading the changes or modifications made to a local repository to the remote server usually on github. 
 
 ## Question 14
 
 What is Github?
 
-A place for us to store and share our code.
+A distributed version control system for developers
```

Comment:
Hi David, i think the question 14 the right answer  git is  version control not for github

Reply:
> Hi David, i think the question 14 the right answer git is version control not for github

Hi Thanks for pointing this out. I will amend my answer accordingly!